frontend/src/index.css

```diff
@@ -1,6 +1,11 @@
 /* adds Inter & JetBrains Mono */
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800&family=JetBrains+Mono:wght@100;200;300;400;500;600;700&display=swap');
 
+@font-face {
+  font-family: lausanne;
+  src: url(./assets/fonts/Lausanne-500.otf);
+}
+
```

Comment:
Are we only supporting 500 weight

Reply:
This font is only a placeholder for the new header logo which only requires 500.  I think Megan is making an SVG which will replace this.

Reply:
Okay, are we removing this font afterwards?  I noticed there was a `.otf` file for 300 weight as well